frontend/src/components/modalContent/AddIncidentModal.js

```diff
@@ -157,7 +162,6 @@ const StepForms = (
         initialValues={props.initialValues}
         incidentDeterminationOptions={props.incidentDeterminationOptions}
         followupActionOptions={props.followupActionOptions}
-        incidentStatusCodeOptions={props.incidentStatusCodeOptions}
         hasFatalities={props.addIncidentFormValues.number_of_fatalities > 0 || false}
```

Comment:
was this an intentional deletion?

Reply:
Doesn't look like it, the Form still has this as a parameter.

Reply:
I did not, I must have selected an extra line when removing the unnecessary passing of categories that I added then removed here. Thank you for noticing! Added back.